Classes/Format/Mods.php

```diff
@@ -173,11 +173,16 @@ private function getAuthorFromXml(array $authors, int $i): void
                 $k++;
             }
             ksort($name);
+
             $this->metadata['author'][$i] = trim(implode(', ', $name));
         }
-        // Append "valueURI" to name using Unicode unit separator.
+
         if (isset($authors[$i]['valueURI'])) {
-            $this->metadata['author'][$i] .= pack('C', 31) . (string) $authors[$i]['valueURI'];
+            $displayName = $this->metadata['author'][$i];
+            $this->metadata['author'][$i] = [
+                'name' => $displayName,
+                'url' => (string) $authors[$i]['valueURI']
+            ];
```

Comment:
How does this affect the indexing of author names? You adjusted the templates for display, but I don't see any changes to the indexing routines. Are those not affected?

Reply:
Thanks for comment. I didn't consider this case. I will look into it.